service-sdk/14.0.0/migrate-to-open-api/message-service-spec-raml/pom.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>backbase-spec-starter-parent</artifactId>
+        <groupId>com.backbase.buildingblocks</groupId>
+        <version>13.0.0</version>
```

Comment:
Sorry, just checking, is this meant to be 14.0.0? thanks

Reply:
14.0.0 doesn't have the plugin referenced in this pom, hence my question in the PR description

Reply:
RAML specs are no longer referenced in Community.

Reply:
I'll get a list together of what projects can be kept/ deleted

Reply:
migrate-to-open-api directory can be deleted, rest are still used by Community